src/engine/engine.js

```diff
@@ -12,7 +11,7 @@ import { Physics } from "./physics/physics";
 import { XRInput } from "./xrinput"
 
 // editor camera
-const camera = new PerspectiveCamera(75, window.innerWidth / window.innerHeight, 0.1, 10000);
+export const camera = new PerspectiveCamera(75, window.innerWidth / window.innerHeight, 0.1, 10000);
```

Comment:
Did this to easily access the camera in my scenes.

Reply:
Capitalize before checkin

Reply:
Incorporated into PongXR